tests/clone_test.go

```diff
@@ -874,3 +729,149 @@ func createSourceVM(options ...libvmi.Option) (*virtv1.VirtualMachine, error) {
 	virtClient := kubevirt.Client()
 	return virtClient.VirtualMachine(vm.Namespace).Create(context.Background(), vm, v1.CreateOptions{})
 }
+
+func createSnapshot(vmName, vmNamespace string) *snapshotv1.VirtualMachineSnapshot {
```

Comment:
Can we reduce `createSnaphot` and `waitSnapshotReady` into a single helper? I only see them used together.

Reply:
Maybe use a fluent interface here?

Reply:
I kinda prefer the "function one one thing" approach. But I'm not gonna insist on it - you call .